### DISCUSSION TITLE
Whispering !

### DISCUSSION THREAD

**Discussion by EricB1978** [stoatchat/discussions#753] [Feature Suggestions]
Let us whisper through the different voice channels just as we would with teamspeak!
We're all waiting for this essential feature for MMO games.

**Comment by SamuraiWaffle** [comment]
This is so useful for managing big groups, please let this happen

**Comment by IAmTomahawkx** [comment]
You'll need to actually explain the feature you're looking for. I don't know what whispering means.

**Reply by SamuraiWaffle** [reply]
Having a push to talk button you can press so that you only speak to a specific person or group of people.

So say for example you use push to talk, you have one button that talks to everyone, one button that only talks to group A, one button that only talks to group B, etc...

Its something Teamspeak has and is super useful when you are playing with large groups.

**Reply by ThreeCrown** [reply]
Omg yes! This is a must have feature! Guilded attempted this, but was a failure of a platform. But it is coveted by big communities and a reason that a lot of big gaming communities still use TS/Mumble.
Basically you can granularly set up Whisper groups and set that whisper to a hotkey.
Then you are able to broadcast your voice to different Channels, Roles, or individuals, and any combination of the following there in.
Say I am in Channel 1, and want to talk to All Admins. Press a hotkey and bam they hear me. And can reply back and only I hear it. Or if they have the hotkey set.

Or in another Scenario. I want to talk to all Group Leaders in Channel 3 & Channel 4. 

So many configurations and usages for this. It's especially useful for extremely large gaming communities. I am moving my 2,000+ community back to TS for this reason in light of recent Discord events, but if you had a feature like this, your platform would be far easier to get everyone over seamlessly and with less complaints.

**Comment by ThreeCrown** [comment]
Omg yes! This is a must have feature! Guilded attempted this, but was a failure of a platform. But it is coveted by big communities and a reason that a lot of big gaming communities still use TS/Mumble.
Basically you can granularly set up Whisper groups and set that whisper to a hotkey.
Then you are able to broadcast your voice to different Channels, Roles, or individuals, and any combination of the following there in.
Say I am in Channel 1, and want to talk to All Admins. Press a hotkey and bam they hear me. And can reply back and only I hear it. Or if they have the hotkey set.

Or in another Scenario. I want to talk to all Group Leaders in Channel 3 & Channel 4.

So many configurations and usages for this. It's especially useful for extremely large gaming communities. I am moving my 2,000+ community back to TS for this reason in light of recent Discord events, but if you had a feature like this, your platform would be far easier to get everyone over seamlessly and with less complaints.
